ocis-pkg/log/log.go

```diff
@@ -58,7 +58,7 @@ func NewLogger(opts ...Option) Logger {
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
 		mlog.SetLevel(mlog.LevelTrace)
 	default:
-		zerolog.SetGlobalLevel(zerolog.InfoLevel)
+		zerolog.SetGlobalLevel(zerolog.ErrorLevel)
 		mlog.SetLevel(mlog.LevelInfo)
```

Comment:
Should this also be changed to the error level?

Reply:
:+1: